netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -322,6 +322,16 @@ public ConnectionClientTransport newClientTransport(
       }
       ProtocolNegotiator negotiator = protocolNegotiator != null ? protocolNegotiator :
           createProtocolNegotiator(authority, negotiationType, sslContext);
+      String proxy = System.getenv("GRPC_PROXY_EXP");
```

Comment:
Sorry, but it isn't immediately obvious to me what does the `_EXP` suffix stand for?